citesphere/src/main/webapp/WEB-INF/views/auth/group/item.jsp

```diff
@@ -250,129 +257,222 @@
-        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-        <h4 class="modal-title">Parse Reference</h4>
-      </div>
-      <div class="modal-body">
-        <p id="ref-unparsed"></p>
-        <table class="table table-striped">
-        <tr>
-            <td width="30%">Author string:</td>
-            <td id="parsed-ref-authors"></td>
-        </tr>
-        <tr>
-            <td>Title:</td>
-            <td id="parsed-ref-title"></td>
-        </tr>
-        <tr>
-            <td>Year:</td>
-            <td id="parsed-ref-year"></td>
-        </tr>
-        <tr>
-            <td>Identifier:</td>
-            <td id="parsed-ref-ident"></td>
-        </tr>
-        <tr>
-            <td>First page:</td>
-            <td id="parsed-ref-first-page"></td>
-        </tr>
-        <tr>
-            <td>Last page:</td>
-            <td id="parsed-ref-last-page"></td>
-        </tr>
-        <tr>
-            <td>Volume:</td>
-            <td id="parsed-ref-volume"></td>
-        </tr>
-        <tr>
-            <td>Source:</td>
-            <td id="parsed-ref-source"></td>
-        </tr>
-        <tr>
-            <td>Publication type:</td>
-            <td id="parsed-ref-publication-type"></td>
-        </tr>
-        <tr>
-	        <td colspan="2">
-		        
-            </td>
-        </tr>
-        <tr>
-            <td>Reference ID:</td>
-            <td id="parsed-ref-reference-id"></td>
-        </tr>
-        <tr>
-            <td>Reference label:</td>
-            <td id="parsed-ref-reference-label"></td>
-        </tr>
-        <tr>
-            <td>Citation ID:</td>
-            <td id="parsed-ref-citation-id"></td>
-        </tr>
-        <tr>
-            <td>Reference as found:</td>
-            <td id="parsed-ref-ref-raw"></td>
-        </tr>
-        </table>
-      </div>
-      <div class="modal-footer">
-        <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
-      </div>
-    </div><!-- /.modal-content -->
-  </div><!-- /.modal-dialog -->
-</div><!-- /.modal -->
+<div class="modal fade" id="deleteModal" tabindex="-1" role="dialog"
+	aria-labelledby="deleteModalLabel" aria-hidden="true">
+	<div class="modal-dialog" role="document">
+		<div class="modal-content">
+
+			<div class="modal-body">Are you sure you want to delete
+				Citation?</div>
+			<div class="modal-footer">
+
+				<button type="button" class="btn btn-secondary" data-dismiss="modal">No</button>
+				<button id="delete" type="button" class="btn btn-primary">Yes</button>
+
+			</div>
+		</div>
+	</div>
+</div>
+
+<div class="modal fade" id="loadModal" tabindex="-1" role="dialog"
+	aria-labelledby="loadModalLabal" aria-hidden="true">
+	<div class="modal-dialog" role="document">
+		<div class="modal-content">
+
+			<div class="modal-body">
+
+				<div>
+					<span><i class="fas fa-spinner text-center"></i> Deleting...</span>
```

Comment:
the spinner should spin

Reply:
Before we discussed that the spinner icon would be fine.

Reply:
maybe I misunderstood you, but the spinner should spin. It should be just one more css class, so should be easy to do.